### ISSUE TITLE
Alt text needed for images

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/iguide-circuit-breaker#68]
I used IBM internal tool Dynamic Assessment Plugin (DAP) to find the accessibility violations.  There are three violations for `Images must have an alt attribute, and must be the empty string if decorative.` 

Page with violations:
https://openliberty.io/guides/circuit-breaker.html

![image](https://user-images.githubusercontent.com/31117513/33493037-8d139b2a-d684-11e7-9e32-f3be8105dff9.png)
